src/coprocessor/util.rs

```diff
@@ -134,6 +134,62 @@ pub fn get_pk(col: &ColumnInfo, h: i64) -> Datum {
     }
 }
 
+use std::time::Duration;
+use std::sync::atomic::{AtomicIsize, Ordering};
+
+// RuntimeStats collects one executor's execution info.
+#[derive(Debug)]
+pub struct RuntimeStats {
+    // executor consume time.
+    pub consume: AtomicIsize,
```

Comment:
I think it is not necessary to use atomic variables here, since executors of a single task won't be executed in parallel.